cmd/maintenance.go

```diff
@@ -18,10 +18,10 @@ func (d DeisCmd) MaintenanceInfo(appID string) error {
 		return err
 	}
 
-	if *appSettings.Maintenance {
-		d.Println("Maintenance mode is on.")
+	if appSettings.Maintenance == nil || !*appSettings.Maintenance {
+		d.Println("Maintenance mode is disabled.")
```

Comment:
needs to be on/off for this to pass CI. For routing it makes sense, but since it's `deis maintenance:on` and `deis maintenance:off` it should say "it's on/off".